portal/models/extension.py

```diff
@@ -26,16 +26,22 @@ class CCExtension(Extension, metaclass=ABCMeta):
     def children(self):  # pragma: no cover
         pass
 
-    def as_fhir(self, include_empties=True):
+    def as_fhir(self, include_empties=True, include_inherited=False):
         if self.children.count():
             return {
                 'url': self.extension_url,
                 'valueCodeableConcept': {
                     'coding': [c.as_fhir() for c in self.children]}
             }
-        # Return valid empty if none are currently defined and requested
         if include_empties:
+            # Return valid empty if none are currently defined and requested
             return {'url': self.extension_url}
+
+        if include_inherited:
+            # TODO in the organization instance, need a `parent()`
```

Comment:
Not clear to me the impact of this remaining a TODO...

Reply:
This method is located in the `CCExtension` class, which is responsible for serialization of CodeableConcepts.  Its only use in organizations is `LocaleExtension`.  In other words, until this todo is tackled, an organization's locales will only show in the FHIR format if they're defined at the requested org's level - we're not yet supporting `include_inherited_attributes` for organization locales.